src/main/java/com/breskul/bibernate/util/Pair.java

```diff
@@ -0,0 +1,8 @@
+package com.breskul.bibernate.util;
+
+public record Pair<L, R>(L left, R right) {
```

Comment:
For readability, I advise using more specific wrapper classes. `Pair` is really hard to read and understand what is going on.